### PR TITLE
no leading space allowed in search input fields

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
#### Which User story does this relate to (link)?
[COG-1422](https://osi-cwds.atlassian.net/browse/COG-1422)

#### Brief feature description
- No leading space in the search fields

### CODE REVIEW

**Comment by Sushmareddy-OSI** [comment]
If we use trim() it wont allow to enter two words in the input field (Ex: 'foo boss' as firstName)

File: app/javascript/views/searchUserList/SearchUserList.jsx [hunk 1]
```diff
@@ -25,17 +25,18 @@ class SearchUserList extends PureComponent {
...
     this.props.actions.fetchRolesActions()
     this.props.actions.clearAddedUserDetailActions()
     this.props.actions.setSearch([
-      { field: 'first_name', value: this.props.firstName },
-      { field: 'last_name', value: this.props.lastName },
-      { field: 'email', value: this.props.email },
-      { field: 'racfid', value: this.props.CWSLogin },
-      { field: 'office_ids', value: this.props.officeNames },
-      { field: 'enabled', value: this.props.includeInactive ? '' : true },
+      { field: 'first_name', value: firstName ? firstName.trim() : '' },
```
[the remark this answers is not in the document]
Comment (Nasir2712) [reply app/javascript/views/searchUserList/SearchUserList.jsx]: Plus if you do it in the selector I guess you don't have to repeat this logic everywhere?

[the remark this answers is not in the document]
Comment (Sushmareddy-OSI) [reply app/javascript/views/searchUserList/SearchUserList.jsx]: I agree to move this to selector!!
But to do that we need to refactor all other methods that are associated with searchQuery, also we need both trimmed and non-trimmed values of the search fields to perform other actions 
there is story to address the performance issue of this component.

[the remark this answers is not in the document]
Comment (Nasir2712) [reply app/javascript/views/searchUserList/SearchUserList.jsx]: ok

**Pull request merged**